app/views/admins/log_in.html.erb

```diff
@@ -0,0 +1,20 @@
+<div>
+  <figure class="profile-pic">
+    <img src="https://www.pngarts.com/files/10/Default-Profile-Picture-PNG-Image-Background.png" alt="profile-picture">
+  </figure>
+  <form class="new-user" id="new_user" action="/">
+      <div class="field">
+        <label for="name"></label>
+        <input type="text" name="admin[name]" id="name" placeholder="Type your name">
+      </div>
+      <div class="field">
+        <label for="password"></label>
+        <input type="password" name="admin[password]" id="password" placeholder="Type your password">
+      </div>
+      <div class="actions">
+        <input type="submit" name="commit" value="Log in" class="primary-button">
+      </div>
+  </form>
```

Comment:
If this form is similar to the one you are going to use to sign up, you can export it to an external file.

In fact, if you use some session gems like [Devise](https://github.com/heartcombo/devise) or [Clearance](https://github.com/thoughtbot/clearance), you will see them output the form to an external file with render:

`<%= render "users/form" %>`

Reply:
We are using SASS for our styles, and when we used application.html.erb some styles were overwritten. So we try to create a specific layout for some views and specify the CSS file for this layout.

But we are not sure if this is the best way or the best practice to solve this problem.

Reply:
Yes, as I understand, gems like "Devise" use their own views. I use this file with similar code to "Devise", because the project doesn't have the gem yet and I needed to see the style changes.

When we add the gem, in next issues, this file will be deleted and the style will be used with the Devise views